### PR TITLE
Fix sed command for NodeLocalDNS yaml in IPVS mode.

### PR DESCRIPTION
**Pull request by prameshj**
Verified that the new sed command generates the correct yaml:

`args: [ "-localip", "169.254.20.10", "-conf", "/etc/Corefile", "-upstreamsvc", "kube-dns-upstream" ]`

Fixes issue - https://github.com/kubernetes/kubernetes/issues/101339

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/nodelocaldns.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ If you are using the sample manifest from the previous point, this will require
   * If kube-proxy is running in IPVS mode:
 
     ``` bash
-     sed -i "s/__PILLAR__LOCAL__DNS__/$localdns/g; s/__PILLAR__DNS__DOMAIN__/$domain/g; s/__PILLAR__DNS__SERVER__//g; s/__PILLAR__CLUSTER__DNS__/$kubedns/g" nodelocaldns.yaml
+     sed -i "s/__PILLAR__LOCAL__DNS__/$localdns/g; s/__PILLAR__DNS__DOMAIN__/$domain/g; s/,__PILLAR__DNS__SERVER__//g; s/__PILLAR__CLUSTER__DNS__/$kubedns/g" nodelocaldns.yaml
```
Comment (aojea) [inline content/en/docs/tasks/administer-cluster/nodelocaldns.md:82]: you have the same sed command in line 73

Comment (aojea) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md:82]: is not easier to read the way that is used in the helper script?
https://github.com/kubernetes/kubernetes/blob/6aa683e9cf9bfa9069a0085eb8a1756fc0678e07/cluster/gce/gci/configure-helper.sh#L2596-L2598

Comment (prameshj) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md:82]: line 73 is for iptables, in that case - __PILLAR__DNS__SERVER__ and __PILLAR__CLUSTER__DNS__ are replaced by valid variables. So it is fine.

In IPVS mode, "__PILLAR__DNS__SERVER__" is replaced by empty string, so the comma needs to be removed too.

Comment (prameshj) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md:82]: Just 3 separate sed commands?

Comment (aojea) [reply content/en/docs/tasks/administer-cluster/nodelocaldns.md:82]: yep, onliners are hard to parse to me, but forget that comment, maybe it is me

**Review by irvifa — COMMENTED** [review]
/approve
